packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -752,6 +752,35 @@ describeCompat("Container", "NoCompat", (getTestObjectProvider) => {
 		},
 	);
 
+	describe("0x314 assert", () => {
+		it("Closing container", async () => {
+			const container = await createConnectedContainer();
+			container.deltaManager.on("disconnect", () => {
+				// Assert 0x314 would appear in "after each" unexpected errors (see DeltaManager ctor)
```

Comment:
Not sure what I'm supposed to see in the DM or Container ctor, can you be more specific here? Or could be sufficient to just leave that pointer out.